lib/state/notes_state.dart

```diff
@@ -159,6 +159,12 @@ class NotesModel extends ChangeNotifier {
       _playerModel.stopRecording();
       return;
     }
+    if (delete) {
+      _playerModel.stopRecording();
+      Sentry.addBreadcrumb(
+          Breadcrumb(message: "Not saving note", timestamp: DateTime.now()));
+      return;
```

Comment:
Note to self (or feel free to do this) - actually delete the file that results. Check out the delete file method for details.